src/resources/language/_language-fr.yml

```diff
@@ -51,3 +51,22 @@ crossref-eq-prefix: "Équation"
 crossref-lof-title: "Liste des Figures"
 crossref-lot-title: "Liste des Tables"
 crossref-lol-title: "Liste des Listings"
+
+listing-page-order-by: "Commander par"
+listing-page-order-by-default: "Présélectionné"
+listing-page-order-by-date-asc: "Le plus récent"
+listing-page-order-by-date-desc: "Le plus ancien"
+listing-page-order-by-number-desc: "Descendant"
+listing-page-order-by-number-asc: "Ascendant"
+listing-page-field-date: "Date"
+listing-page-field-title: "Titre"
+listing-page-field-description: "La description"
```

Comment:
Not sure you need the `La` here. 

`Description` in English would juts be `Description` in french too. 

But I am lacking context of where this is used in the website.

Reply:
It would be used, for example, as a heading for a column of a table displaying the 'description' of a document (e.g. the abstract, if provided or the first paragraph). I think your suggestion is correct.

I'm planning to get some documentation put together than throw out some formal request for help with these translations - but very very happy to have you looking at this right away!!! Thank you!!!

Reply:
Great. For anything French related (or windows specific 😉 ) do not hesitate to ping me! 

I happened to have check notification right after you submitted this PR 😄